src/css/styles.css

```diff
@@ -6,6 +21,214 @@
 
 body {
   font-family: sans-serif;
+  background-color: var(--light-gray);
+}
+
+nav {
+  background-color: var(--white);
+  padding: 1.43rem var(--container-padding);
+  border-bottom: solid 1px var(--white-two);
+}
+
+.nav-title {
+  font-size: 1.87rem;
+  color: var(--light-blue);
+  font-family: 'Montserrat', sans-serif;
+  margin-right: 2.93rem;
+}
+
+.nav-links {
+  column-gap: 2.56rem;
+}
+
+.nav-link {
+  color: var(--black);
+  font-size: 0.81rem;
+  font-weight: normal;
+  text-decoration: none;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.nav-link:hover {
+  opacity: 0.5;
+  transition: all 0.2s ease-in-out;
+}
+
+.nav-user {
+  border: 1px solid var(--white-two);
+  padding: 0.87rem 0.87rem 0.8rem 0.87rem;
+  border-radius: 50%;
+}
+
+.nav-user > img {
+  filter: invert(40%) sepia(75%) saturate(3235%) hue-rotate(189deg) brightness(103%) contrast(107%);
+}
+
+.bg-white {
+  background-color: var(--white);
+}
+
+.text-black {
+  color: var(--black);
+}
+
+.text-blue {
+  color: var(--light-blue);
+}
+
+.opacity-5 {
+  opacity: 0.5;
+}
+
+.flex {
+  display: flex;
+}
+
+.justify-between {
+  justify-content: space-between;
+}
+
+.items-center {
+  align-items: center;
+}
+
+.books-list {
+  display: flex;
+  flex-direction: column;
+  row-gap: 0.938rem;
+}
```

Comment:
- [ ] [Optional] I can see that you write custom styles and add them to your components elements, My recommendation to you is to use libraries like Bootstrap or tailwind to do that because it will save you a lot of time spent on writing the custom class styling yourself.